org.eclipse.richbeans.widgets/src/org/eclipse/richbeans/widgets/scalebox/NumberBox.java

```diff
@@ -95,6 +95,9 @@
 	private ActiveMode activeMode = ActiveMode.SET_VISIBLE_AND_ACTIVE;
 	private String boundsKey;
 	
+	// FIXME Widgets should not have a reference to the controller.
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1134)